### PR TITLE
Dutch translation

### PR DESCRIPTION
**Pull request by spmvanmierlo**

### CODE REVIEW

**Comment by spmvanmierlo** [comment]
Do I need to change something due the fact travis-ci fails?

File: Umweltzone/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+        voertuig. Voertuigen die niet voldoen aan een bepaalde dieseluitlaatemissie
+        standaard zijn beperkt om een ​​DPZ in te voeren. Het doel van de inreisbeperkingen is
+        om de luchtkwaliteit in steden te verbeteren.
+        Fijne deeltjes moeten worden verminderd om de gezondheid van de burger te behouden. &#8212;
+        Naast de kaart en een lijst met steden is er een detailscherm dat
+        bevat individuele informatie voor de geselecteerde stad.
+        Veelgestelde vragen zijn te vinden via de
+        &#8220; FAQ &#8221; menu onderdeel. Bovendien zijn er links naar websites die
+        meer informatie bieden over LEZ, DPZ en gerelateerde onderwerpen.
+    </string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-nl/strings.xml]: Please check preview of the longer texts block in Android Studio in `activity_about.xml`. There is a couple of blanks in the texts.

File: Umweltzone/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+        van gescande kaarten kan niet worden gegarandeerd dat de zone precies grenst
+        overeenkomen met het pad in de realiteit. Zonegegevens en coördinaten verschillen per bron,
+        kwaliteit en precisie in het digitalisatieproces.
+        Bovendien is er geen automatisme om op de hoogte te worden gebracht van aangebrachte wijzigingen
+        door de steden. Zonegegevens worden handmatig bijgewerkt en zijn sterk afhankelijk van de
+        feedback van vriendelijke gebruikers zoals jij.
+        Om deze redenen kunnen we niet garanderen voor een perfecte visualisatie en
+        wij zijn niet aansprakelijk voor kosten of straffen. We raden ten zeerste aan om altijd
+        vertrouw op het verkeersbord in de betreffende steden.
+    </string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-nl/strings.xml]: Please check preview of the longer texts block in Android Studio in `activity_about.xml`. There is a couple of blanks in the texts. Scroll down in the preview to see the text.

File: Umweltzone/src/main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+
+    <!-- AboutActivity / License -->
+    <string name="appinfo_license_title">Licenties</string>
+    <string name="appinfo_license_content">
+        De broncode van de Umweltzone-applicatie valt onder de GPLv3-licentie.
+        De geometriecoördinaten uit OpenStreetMap zijn in licentie gegeven onder de
+        Open Database License (ODbL) v1.0. Andere geometriecoördinaten zijn gelicentieerd als
+        gedefinieerd door de genoemde gegevensbronnen. Afbeeldingen en teksten zijn gelicentieerd onder
+        de Creative Commons Attribution 3.0 Unported-licentie.
+    </string>
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-nl/strings.xml]: Please check preview of the longer texts block in Android Studio in `activity_about.xml`. There is a couple of blanks in the texts. Once you scroll down in the preview you can see the text being rendered.

**Comment by johnjohndoe** [comment]
> Do I need to change something due the fact travis-ci fails?

Please note that you can check for the error found by Travis CI yourself. Click on `Details` and scroll to the end of the console log.

image

**Comment by johnjohndoe** [comment]
:heavy_check_mark: Instrumentation tests executed successfully with device set to Dutch. Tested on Nexus 9, Android 7.1.1.

**Comment by johnjohndoe** [comment]
@spmvanmierlo Thanks a lot for your contribution. :+1: I appreciate!

**Comment by johnjohndoe** [comment]
Hey @spmvanmierlo. If you have time there is a few strings to translate [here](https://github.com/Umweltzone/roadsigns/issues/6).

**Pull request merged**